File: VB/WindowsApplication3/MyRibbonCaptionViewInfo.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Windows.Forms
Imports DevExpress.Skins
Imports DevExpress.XtraBars.Ribbon.ViewInfo

Namespace DXSample
	Public Class MyRibbonCaptionViewInfo
		Inherits RibbonCaptionViewInfo
		Public Sub New(ByVal viewInfo As RibbonViewInfo)
			MyBase.New(viewInfo)
		End Sub

		Public Overrides Function CalcGlassFormTopMargin() As Integer
			Dim top As Integer = MyBase.CalcGlassFormTopMargin()
			Return top + Ribbon.CustomSpace
		End Function
		Protected Shadows ReadOnly Property Ribbon() As MyRibbonControl
			Get
				Return TryCast(MyBase.Ribbon, MyRibbonControl)
			End Get
		End Property
	End Class
End Namespace

File: CS/WindowsApplication3/MyRibbonCaptionViewInfo.cs
using System;
using System.Collections.Generic;
using System.Windows.Forms;
using DevExpress.Skins;
using DevExpress.XtraBars.Ribbon.ViewInfo;

namespace DXSample {
    public class MyRibbonCaptionViewInfo : RibbonCaptionViewInfo
    {
        public MyRibbonCaptionViewInfo(RibbonViewInfo viewInfo) : base(viewInfo) { }

        public override int CalcGlassFormTopMargin()
        {
            int top = base.CalcGlassFormTopMargin();
            return top + Ribbon.CustomSpace;
        }
        protected new MyRibbonControl Ribbon
        {
            get { return base.Ribbon as MyRibbonControl; }
        }
    }
}